packages/framework/collabspaces/src/contracts.ts

```diff
@@ -83,5 +94,27 @@ export interface IEfficientMatrixTest {
 	isAttached: boolean;
 
 	// Returns a structure with various debug info about the cell
-	getCellDebugInfo(row: number, col: number): Promise<{ channel?: ICollabChannelCore }>;
+	getCellDebugInfo(
+		row: number,
+		col: number,
+	): Promise<{ channel?: ICollabChannelCore; channelId: string; rowId: string; colId: string }>;
+
+	getReverseMapsDebugInfo(
+		rowId?: string | undefined,
+		colId?: string | undefined,
+	): ReverseMapsDebugInfoType;
```

Comment:
API looks a bit weird - it returns data about specific cell, but also about whole matrix. I'd break it into two APIs.

Reply:
split into 2